hydrosdk/cluster.py

```diff
@@ -11,30 +11,33 @@ class Cluster:
     Cluster responsible for server interactions
     """
     @staticmethod
-    def connect(address):
+    def connect(http_address, grpc_address=None):
```

Comment:
❌  Could you please add docstring for grpc_address

Reply:
⚠️ `Cluster.connect` method tries to establish contact with the server to ensure that addresses are good to go. Could we implement such connectivity test for grpc?

Reply:
Added connectivity test